draft-irtf-panrg-what-not-to-do.md

```diff
@@ -890,6 +925,53 @@ Although {{RFC6437}} recommended that endpoints should by default choose uniform
 
 A growth in the use of encrypted transports, (e.g. QUIC {{QUIC-WG}}) seems likely to raise similar issues to those discussed above and could motivate renewed interest in utilizing the flow label.
 
+## Explicit Congestion Notification (ECN) {#ecn}
+
+The suggested references for Explicit Congestion Notification (ECN) are:
+
+- Recommendations on Queue Management and Congestion Avoidance in the Internet {{RFC2309}}
+- A Proposal to add Explicit Congestion Notification (ECN) to IP {{RFC2481}}
+- The Addition of Explicit Congestion Notification (ECN) to IP {{RFC3168}}
+- Implementation Report on Experiences with Various TCP RFCs {{vista-impl}}, slides 6 and 7
+- Implementation and Deployment of ECN {{SallyFloyd}}
+
+In the early 1990s, the large marjority of Internet traffic used TCP as its transport protocol, but TCP had no way to detect path congestion before the path was so congested that packets were being dropped, and these congestion events could affect all senders using a path, either by "lockout", where long-lived flows monopolized the queues along a path, or by "full queues", where queues remain full, or almost full, for a long period of time.
+
+In response to this situation, "Active Queue Management" (AQM) was deployed in the network. A number of AQM disciplines have been deployed, but one common approach was that routers dropped packets when a threshold buffer length was reached, so that transport protocols like TCP that were responsive to loss would detect this loss and reduce their sending rates. Random Early Detection (RED) was one such proposal in the IETF. As the name suggests, a router using RED as its AQM discipline that detected time-averaged queue lengths passing a threshold would choose incoming packets probabilistically to be dropped {{RFC2309}}.
+
+Researchers suggested that providing "explicit congestion notifications" to senders when routers along the path detected their queues were building, giving them an indication that router queues along the path were building, so that some senders would "slow down" as if a loss had occurred, so that the path queues had time to drain, and the path still had sufficient buffer capacity to accommodate bursty arrivals of packets from other senders. This was proposed as an Experiment in {{RFC2481}}, and standardized in {{RFC3168}}.
```

Comment:
```suggestion
Researchers suggested that providing "explicit congestion notifications" to senders when routers along the path detected their queues were building, so that some senders would "slow down" as if a loss had occurred, the path queues had time to drain, and the path still had sufficient buffer capacity to accommodate bursty arrivals of packets from other senders. This was proposed as an Experiment in {{RFC2481}}, and standardized in {{RFC3168}}.
```